backend/wrapped/serializers.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+from wrapped.models import GlobalStat, IndividualStat, UserStatKey, StatLocation, Page
+
+User = get_user_model()
+
+class MiniUserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ["username", "first_name", "last_name"]
+
+
+class StatLocationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = StatLocation
+        fields = ["IndividualStatKey", "GlobalStatKey", "text_field_name"]
+
+class IndividualStatSerializer(serializers.ModelSerializer):
+    class Meta: 
+        model = IndividualStat
+        fields = ["user", "stat_key", "stat_value", "semester"]
+
+class GlobalStatSerializer(serializers.ModelSerializer):
+    class Meta: 
+        model = GlobalStat
+        fields = ["stat_key", "stat_value", "semester"]
+
+class UserStatKeySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserStatKey
+        fields = ["stat_key"]
+
+class PageSerializer(serializers.ModelSerializer):
```

Comment:
You're not really using the other serializers since you're just serializing it and immediately reformatting it, so might as well not use it. 

The way you should be thinking about serializers is that everytime you have a queryset and then you say to yourself "I wish I could have this queryset in this particular format" you should make a serializer.